web3.js/src/connection.ts

```diff
@@ -2407,6 +2407,32 @@ export class Connection {
     }
   }
 
+  /**
+   * Fetch all the account info for multiple accounts specified by an array of public keys
+   */
+  async getMultipleAccountsInfo(
+    publicKeys: PublicKey[],
+    commitment?: Commitment,
+  ): Promise<AccountInfo<Buffer>[] | null> {
+    const keys = publicKeys.map(key => key.toBase58());
+    try {
```

Comment:
This outer try block isn't necessary. (based on convention established by other calls and redundant throw if 'error' exists in res)